helpers/icon_in_div_changed.py

```diff
@@ -0,0 +1,15 @@
+from pages.lobby_page import LobbyPage
+
+
+class IconInDivChanged(object):
```

Comment:
Do we really need a separate file for this class?
Can it be a method inside lobby_page, with more meaningful name like "status" or similar?

Reply:
We use it as a condition for wait and it should be callable object. But you are right, the functionality is related only to lobby_page, so the class was moved to lobby_page.